backend/pollpall_project/session/models.py

```diff
@@ -1,8 +1,41 @@
 from django.db import models
+from .models import Question
 
 class Session(models.Model):
     session_id = models.IntegerField(primary_key=True)
     session_label = models.CharField(max_length=255)
 
     def __str__(self):
-        return f"{self.session_id}: {self.session_label}"
\ No newline at end of file
+        return f"{self.session_id}: {self.session_label}"
+
+# TODO
+# By using ForeignKey, each question should be automatically connected to the poll
+# that fetches it. Each response is also connected to it's respective
+# question, so we shouldn't need a list of responses.
+class Poll(models.Model):
+    poll_id = models.IntegerField(primary_key=True) 
+    session = models.ForeignKey('Session', on_delete=models.CASCADE)
+    question_id = models.IntegerField(default=-1)
+    related_question = models.ForeignKey('Question', on_delete=models.CASCADE, related_name='related_polls')
+    is_accepting_answers = models.BooleanField(default=False)
+    
+    def __str__(self):
+        return f"{self.poll_id}: {self.session}: {self.question_id}: {self.related_question}: {self.is_accepting_answers}"
+
+class Question(models.Model): # Just holds a question ID and the question
```

Comment:
In python, we can use Docustrings for documentation
```suggestion
class Question(models.Model):
""" 

Holds the question we are asking 

    Attributes:
        question_id (int): The primary key of the model
        prompt (str): The question we are asking
        poll (Poll): The poll this question comes from
"""
```